net/http/src/THttpLongPollEngine.cxx

```diff
@@ -120,12 +198,22 @@ Bool_t THttpLongPollEngine::PreviewData(THttpCallArg &arg)
 /// Normally requests from client does not replied directly
 /// Therefore one can use it to send data with it
 
-void THttpLongPollEngine::PostProcess(THttpCallArg &arg)
+void THttpLongPollEngine::PostProcess(std::shared_ptr<THttpCallArg> &arg)
 {
-   if ((fQueue.size() > 0) && arg.IsContentType("text/plain") &&
-       (arg.GetContentLength() == (Long_t)strlen(gLongPollNope)) &&
-       (strcmp((const char *)arg.GetContent(), gLongPollNope) == 0)) {
-      arg.SetContent(fQueue.front().c_str());
-      fQueue.erase(fQueue.begin());
+   if (arg->IsText() && (arg->GetContentLength() == (Long_t)strlen(gLongPollNope)) &&
```

Comment:
Calling this is not an error if arg is not text?

Reply:
Method called for any request after user process it.
And if user does not assign any new values - one can use it for something else